swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -522,13 +479,13 @@ function initializeHttpClient(string url) returns http:Client|error => new (url)
 
 function initializeEmailClient() returns email:SmtpClient|error => new (host, username, password);
 
-service /healthcare on new http:Listener(port) {
```

Comment:
Not related to this line. 

```
function initializeHttpClient(string url) returns http:Client|error => new (url);
function initializeEmailClient() returns email:SmtpClient|error => new (host, username, password);
```

We have decided to remove the function approach to initialize the clients (hospitalServiceBackend, paymentBackend).